src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball extends Lottery {
+    private String color;
+    private int number;
+
+    protected void setColorNumber() {
+        this.color = new ColorSupplier().getRandomColor();
+        this.number = new Random().nextInt(100);
```

Comment:
Instead of creating a new instance of Random every time, consider having a single instance as a class-level variable, and use a constant for the bound '100'.